crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -415,6 +415,7 @@ spec:
       refresh_interval: "60s"
     validations:
       ignore: ["KIA1201"]
+      skip_wildcard_gateway_hosts: false
```

Comment:
Is there a reason why we add a separate config and not just tell people to add  KIA301 to the ignore list above? This seems confusing to allow to ignore validations by ID but then add a setting for a specific setting.

Reply:
There is a community user request to keep the validations of KIA0301 but to ignore them for wildcard only hostnames: https://github.com/kiali/kiali/issues/5711
As Istio itself is not conflicting with wildcard hosts but leaving them as a last Gateway to be used.

Reply:
Adding a new validation code would make a confusion in two quite similar codes, so I decided to make a small change in existing one. Agree to have it well documented in crd schema.